src/pkg/backup/details/details.go

```diff
@@ -14,10 +14,11 @@ import (
 	"github.com/alcionai/corso/src/pkg/path"
 )
 
-type FolderEntry struct {
+type folderEntry struct {
```

Comment:
Minor change - `FolderEntry` is pkg internal now. Apologies for the noise related to this in the PR